lms/product/plugin/misc.py

```diff
@@ -111,16 +117,27 @@ def is_speed_grader_launch(self, _request) -> bool:  # pragma: nocover
 
     @staticmethod
     def _assignment_config_from_assignment(assignment: Assignment) -> AssignmentConfig:
-        return {
-            "document_url": assignment.document_url,
-            "group_set_id": assignment.extra.get("group_set_id"),
-        }
+        """Get the configuration of an assignment from our DB."""
+        config = AssignmentConfig(
+            document_url=assignment.document_url,
+            group_set_id=assignment.extra.get("group_set_id"),
+        )
+        if auto_grading_config := assignment.auto_grading_config:
```

Comment:
For assignments that are configured in our DB, look for the autograding config there.